packages/react-components/react-tags/stories/src/TagGroup/TagGroupSelect.stories.tsx

```diff
@@ -0,0 +1,160 @@
+ */
+const useResetExample = (visibleTagsLength: number) => {
+  const resetButtonRef = React.useRef<HTMLButtonElement>(null);
+  const firstTagRef = React.useRef<HTMLButtonElement>(null);
+
+  const prevVisibleTagsLengthRef = React.useRef<number>(visibleTagsLength);
+  React.useEffect(() => {
+    if (visibleTagsLength === 0) {
+      resetButtonRef.current?.focus();
+    } else if (prevVisibleTagsLengthRef.current === 0) {
+      firstTagRef.current?.focus();
+    }
+
+    prevVisibleTagsLengthRef.current = visibleTagsLength;
+  }, [visibleTagsLength]);
+
+  return { firstTagRef, resetButtonRef };
+};
+
+const DismissWithInteractionTags = () => {
+  const [visibleTags, setVisibleTags] = React.useState(initialTags);
+  const [seletedTags, setSelectedTags] = React.useState<Array<TagValue> | undefined>([]);
+
+  const removeItem: TagGroupProps['onDismiss'] = (_e, { value }) => {
+    if (seletedTags?.includes(value)) {
+      setSelectedTags(seletedTags?.filter(tag => tag !== value));
+    }
+    setVisibleTags([...visibleTags].filter(tag => tag.value !== value));
+  };
+
+  const resetItems = () => setVisibleTags(initialTags);
+  const selectItem: TagGroupProps['onTagSelect'] = (_e, { value }) => {
+    if (!seletedTags) {
+      return;
+    }
+    if (seletedTags.includes(value)) {
+      setSelectedTags(seletedTags.filter(tag => tag !== value));
+    } else {
+      setSelectedTags([...seletedTags, value]);
+    }
+  };
+
+  const { firstTagRef, resetButtonRef } = useResetExample(visibleTags.length);
+
+  const styles = useStyles();
+
+  return (
+    <>
+      <p>Selected values: {seletedTags?.join(', ')}</p>
+      {visibleTags.length !== 0 && (
+        <TagGroup
+          onTagSelect={selectItem}
+          onDismiss={removeItem}
+          aria-label="Tag group with Dismissable Multiselect Tag"
+        >
+          {visibleTags.map((tag, index) => (
+            <InteractionTag value={tag.value} key={tag.value}>
+              <InteractionTagPrimary hasSecondaryAction ref={index === 0 ? firstTagRef : null}>
+                {tag.children}
+              </InteractionTagPrimary>
+              <InteractionTagSecondary aria-label="remove" />
+            </InteractionTag>
+          ))}
+        </TagGroup>
+      )}
+      <Button
+        onClick={resetItems}
+        ref={resetButtonRef}
+        disabled={visibleTags.length !== 0}
+        className={styles.resetButton}
+        size="small"
+      >
+        Reset the example
+      </Button>
+    </>
+  );
+};
+
+export const Select = () => {
+  const styles = useStyles();
+  return (
+    <div className={styles.container}>
+      Example with InteractionTag:
+      <WithMultiselectTags />
+      Example with Dismissable InteractionTag:
+      <DismissWithInteractionTags />
+    </div>
+  );
+};
+
+Select.storyName = 'Select';
+Select.parameters = {
+  docs: {
+    description: {
+      story: 'A TagGroup contains a collection of InteractionTag that can be selected.',
+    },
```

Comment:
Hi there, can you add comment in the story that are similar to what you have in API doc so users are aware selected state has no a11y implementation yet? Based on my observation our feature engineers prefer reading stories over api docs.  


>Note: This prop only changes the appearance of the tag at the moment. A future PR will add the integration with TagGroup.

Reply:
And same goes with the other story

Reply:
added those comments to the TagGroup and IntaractionTag stories